protocols/dcutr/tests/lib.rs

```diff
@@ -153,7 +153,7 @@ where
     prelude = "libp2p_swarm::derive_prelude"
 )]
 struct Client {
-    relay: client::Client,
+    relay: client::Behaviour,
```

Comment:
```suggestion
    relay: relay::client::Behaviour,
```

Here too?

Reply:
yeah I think it makes sense across the whole file tbh. Updated!